mos_tests/environment/os_actions.py

```diff
@@ -84,6 +85,14 @@ def __init__(self, controller_ip, user='admin', password='admin',
         self.glance = GlanceClient(endpoint=glance_endpoint,
                                    token=token,
                                    cacert=path_to_cert)
+
+        heat_endpoint = self.keystone.service_catalog.url_for(
```

Comment:
It is not mandatory, but it is deploying by default and there is no way to disable it.